lib/core/BlogSidebar.js

```diff
@@ -13,16 +13,30 @@ const MetadataBlog = require('./MetadataBlog.js');
 
 class BlogSidebar extends React.Component {
   render() {
+    let blogSidebarCount = 5;
+    let blogSidebarTitle = 'Recent Posts';
+    if (this.props.config.blogSidebarCount) {
+      if (this.props.config.blogSidebarCount == 'ALL') {
+        blogSidebarCount = MetadataBlog.length;
+        blogSidebarTitle = 'All Blog Posts';
+      } else {
+        blogSidebarCount = this.props.config.blogSidebarCount;
+      }
+    } else {
+      blogSidebarCount = 5;
```

Comment:
I don't think you need this `else` because you already set this to `5` in the `let above.

Reply:
Nice. Removed.